pkg/component/kubernetes/apiserver/apiserver_suite_test.go

```diff
@@ -9,9 +9,12 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+
+	"github.com/gardener/gardener/pkg/gardenlet/features"
 )
 
 func TestAPIServer(t *testing.T) {
 	RegisterFailHandler(Fail)
+	features.RegisterFeatureGates()
```

Comment:
Was there a specific reason to call `features.RegisterFeatureGates()` after `RegisterFailHandler(Fail)`. I think rest of the occurrences in the codebase do it vice-versa - call `features.RegisterFeatureGates()` before `RegisterFailHandler(Fail)`.

```suggestion
	features.RegisterFeatureGates()
	RegisterFailHandler(Fail)
```